terraform/main.mk

```diff
@@ -3,6 +3,8 @@
 SSH_PUBLIC_KEY ?= $(shell cat ~/.ssh/id_rsa.pub)
 EC2_KEY_PAIR_NAME ?= $(ENV)-$(NAMESPACE)
 ENV_DIR ?= $(INFRA_DIR)/env/$(ENV)
+JSON_OUTPUT_FILE = $(INFRA_DIR)/env/$(ENV)/output.json
+OUTPUT_JSON_BASE64 = $(shell cat ${JSON_OUTPUT_FILE} | $(BASE64))
```

Comment:
`${JSON_OUTPUT_FILE}` is not a make construct, I'd use $(JSON_OUTPUT_FILE)

Reply:
Understand.